rubix-build/pom.xml

```diff
@@ -84,6 +84,13 @@
                                     <pattern>org.apache.thrift</pattern>
                                     <shadedPattern>org.apache.thrift.shaded</shadedPattern>
                                 </relocation>
+                                <relocation>
+                                    <pattern>com.google</pattern>
+                                    <shadedPattern>com.google.shaded</shadedPattern>
+                                    <excludes>
```

Comment:
any reason to exclude classes under this package?

Reply:
It was done in this [PR](https://github.com/qubole/rubix/pull/415), so I added in build module too.

Reply:
got it.